sdk/identity/Azure.Identity/src/Credentials/InteractiveBrowserCredentialOptions.cs

```diff
@@ -64,5 +65,21 @@ public string TenantId
 
         /// <inheritdoc/>
         public bool DisableInstanceDiscovery { get; set; }
+
+        /// <summary>
+        /// Specifies if the public client application should used an embedded web browser
+        /// or the system default browser
+        /// </summary>
+        public bool? UseEmbeddedWebView { get; set; }
+
+        /// <summary>
+        /// The options for using the system OS browser handle interactive authentication.
+        /// </summary>
+        public SystemWebViewOptions SystemBrowserOptions { get; set; }
+
+        /// <summary>
+        /// The options for using the embedded web view for interactive authentication.
+        /// </summary>
+        public EmbeddedWebViewOptions EmbeddedBrowserOptions { get; set; }
```

Comment:
We generally don't expose MSAL types directly in our public API surface. These should be wrapped in a type we own.

Reply:
I believe we can simplify this quite a bit by only exposing an option to control whether the system or embedded browser will be invoked.